### PR TITLE
First draft of a maubot role

### PR DESCRIPTION
**Pull request by Cadair**
This adds https://github.com/maubot/maubot

I have done much of this blind and with copy paste, so I am sure there is a lot to improve!

### CODE REVIEW

**Comment by aaronraimist** [comment]
@paulbdavis also has a maubot role here: https://github.com/dangersalad/matrix-docker-ansible-deploy which has been working for me

(separated on its own branch here just to make it easier to see the changes https://github.com/spantaleev/matrix-docker-ansible-deploy/compare/master...aaronraimist:maubot)

**Comment by Cadair** [comment]
That looks a lot more complete than this one, can it get PRed over?

**Comment by aaronraimist** [comment]
He said he is planning to after he PRs the bridges. The role works, I've been running it for a couple days. He said there was some changes he needed to make with the docker image so he'll need to make a PR with those to maubot.

**Comment by Cadair** [comment]
Well I am very happy to close this in favour of that once it's open.

File: roles/matrix-base/tasks/validate_config.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@
       The `{{ item }}` variable defined in your configuration is not used by this playbook anymore.
       User/group creation is now dynamic. You can remove these variables from your configuration, as they have no effect on anything.
   when: "item in vars"
-  with_items:
-    - 'matrix_user_uid'
-    - 'matrix_user_gid'
+  with_items: []
+          #- 'matrix_user_uid'
+          #- 'matrix_user_gid'
```
Comment (SuperSandro2000) [inline roles/matrix-base/tasks/validate_config.yml:11]: In git is is a good practice to remove such code and later dig trough the history if you need it again.

**Pull request closed without merging**